### PR TITLE
UNT Government Web & Data Archive submission

### PR DESCRIPTION
**Pull request by atesgoral**
Simultaneously submit form data to Google Forms and UNT's GWDA Nomination Tool.
Also, massive refactoring.

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -20,12 +20,11 @@
   },
   "permissions": [
     "tabs",
+    "cookies",
     "http://*/*",
     "https://*/*",
     "clipboardRead",
     "clipboardWrite",
-    "identity",
-    "http://digital2.library.unt.edu/nomination/eth2016/url/",
```
Comment (atesgoral) [inline manifest.json:28]: No need to list specific URLs since we already have wildcards above.

File: manifest.json [hunk 2]
```diff
@@ -20,12 +20,11 @@
   },
   "permissions": [
     "tabs",
+    "cookies",
```
Comment (atesgoral) [inline manifest.json:23]: This is needed to read cookies (the CSRF cookie, specifically).

File: popup.js [hunk 3]
```diff
@@ -132,172 +132,186 @@ function checkUrl() {
...
+function submitToUntGwda(fields) {
+  console.log('Submitting to UNT GWDA');
+
+  var url = 'http://digital2.library.unt.edu/nomination/GWDA/add/';
+
+  var fieldConversion = {
+    url: 'url_value',
+    nominatorName: 'nominator_name',
+    nominatorEmail: 'nominator_email',
+    eventName: 'nominator_institution'
```
Comment (atesgoral) [inline popup.js:220]: Assumed the closest thing to "institution" would be the event name?

File: popup.js [hunk 4]
```diff
@@ -132,172 +132,186 @@ function checkUrl() {
...
-  if ( localStorage.organizationID && localStorage.organizationID !== "null" ) {
-    $( '#organizationID' ).val( localStorage.organizationID );
-  }
-  if ( localStorage.suborgID && localStorage.suborgID !== "null" ) {
-    $( '#suborgID' ).val( localStorage.suborgID );
-  }
-  if ( localStorage.subprimerID && localStorage.subprimerID !== "null" ) {
-    $( '#subprimerID' ).val( localStorage.subprimerID );
-  }
+  checkAgency();
```
Comment (atesgoral) [inline popup.js:312]: Fixed bug: Wasn't doing this check after recovering agency values from localStorage.

File: popup.js (hunk 3 above)
Comment (titaniumbones) [reply popup.js:220]: yeah, that's what I was thinking.

File: popup.js (hunk 4 above)
Comment (titaniumbones) [reply popup.js:312]: :-)

**Comment by titaniumbones** [comment]
Not able to run tests right now -- but this looks good to me.  will merge once we figure out what's wrong with the spreadsheet, I think.

**Comment by atesgoral** [comment]
*bump*

**Comment by sonalranjit** [comment]
@titaniumbones any confirmation on the update of the submission directly to archive.org?

**Comment by titaniumbones** [comment]
submission to archive.org is successful. I'm merging this now -- there are no outstanding errors, right Ates? Just waiting for me to do this.

**Comment by titaniumbones** [comment]
Or did we want to have a conversation about the success criteria?

**Comment by atesgoral** [comment]
No issues that I'm aware of. We need to discuss the success criteria. Currently, if any of the two submissions fails, a failure message is shown.

**Pull request merged**